packages/cli/src/babel-stuff.ts

```diff
@@ -0,0 +1,119 @@
+import * as babel from "@babel/core";
+import path from "path";
+import crypto from "crypto";
+import resolveFrom from "resolve-from";
+
+function normalizeOptions(config: {
+  options: any;
+  passes: any;
+}): babel.TransformOptions & {
+  parserOpts: babel.ParserOptions;
+  generatorOpts: babel.GeneratorOptions;
+} {
+  const {
+    filename,
+    cwd,
+    filenameRelative = typeof filename === "string"
+      ? path.relative(cwd, filename)
+      : "unknown",
+    sourceType = "module",
```

Comment:
is this Babel's default? i thought it was `"unambiguous"` but also it didn't matter for me much so I might have been under a wrong impression

Reply:
The default is "unambiguous" but everything that Preconstruct transforms is a module so I changed this